### PR TITLE
Enable cross-dichroics in cube_build

### PR DESCRIPTION
**Pull request by drlaw1558**
As part of implementing cross-dichroic support in spec2 I realized that in addition to simple reference file updates cube_build also required a few code updates to know how to handle cross-dichroic BAND keywords.  These seemed pretty simple, so this PR implements the necessary changes.  Tested the new code with the new cubepar to be delivered to CRDS and it successfully builds cross-dichroic data cubes in spec2 and standalone cube build, both internal_cal and regular coordinate frames.

Tagging @jemorrison to verify whether I got anything wrong!

### CODE REVIEW

File: CHANGES.rst [hunk 1]
```diff
@@ -1,6 +1,11 @@
 0.18.4 (unreleased)
 ===================
 
+cube_build
+----------
+
+- Added support for cross-dichroic configurations
```
Comment (hbushouse) [inline CHANGES.rst:7]: Add the PR number to the end of the line, e.g. [#5722]

**Review by hbushouse — COMMENTED** [review]
Looks OK to me, but I'm no cube_build expert.

File: CHANGES.rst (hunk 1 above)
Comment (drlaw1558) [reply CHANGES.rst]: Fixed

**Review by jemorrison — APPROVED** [review]
Seems fine.  I just checked CRDS you already have the new cubepars reference file loaded with these new options.
Great.

**Review by hbushouse — APPROVED** [review]

**Pull request merged**